NitroxClient/Communication/Packets/Processors/GhostCrafterItemPickupProcessor.cs

```diff
@@ -9,26 +8,14 @@ namespace NitroxClient.Communication.Packets.Processors
 {
     public class GhostCrafterItemPickupProcessor : ClientPacketProcessor<GhostCrafterItemPickup>
     {
-        private readonly IPacketSender packetSender;
-
-        public GhostCrafterItemPickupProcessor(IPacketSender packetSender)
-        {
-            this.packetSender = packetSender;
-        }
-
         public override void Process(GhostCrafterItemPickup packet)
         {
             GameObject gameObject = NitroxEntity.RequireObjectFrom(packet.GhostCrafterId);
             CrafterLogic crafterLogic = gameObject.RequireComponentInChildren<CrafterLogic>(true);
 
             if (crafterLogic.numCrafted > 0)
             {
-                crafterLogic.numCrafted--;
-
-                if (crafterLogic.numCrafted == 0)
-                {
-                    crafterLogic.Reset();
-                }
+                crafterLogic.Reset();
```

Comment:
Wouldn't this cause an issues if multiple items are crafted but only a few could be picked up (because of inventory space)?